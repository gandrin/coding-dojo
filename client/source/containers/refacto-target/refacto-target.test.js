import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';

import { RefactoTarget } from './index';

describe.only('[Snapshot] RefactoTarget', () => {
  const props = {
    mainActions: {
      doStuff: jest.mock(),
      doSomething: jest.mock()
    },
    main: 'toto'
  };

    const container = renderer.create(<RefactoTarget {...props} />);
    it('should match snapshot :', () => {
      expect(container.toJSON()).toMatchSnapshot();
    });
});
