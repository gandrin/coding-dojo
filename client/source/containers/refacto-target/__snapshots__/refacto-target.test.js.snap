// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Snapshot] RefactoTarget should match snapshot : 1`] = `
<div
  style={
    Object {
      "margin": "10px",
      "position": "relative",
      "textAlign": "center",
    }
  }
>
  toto
  <div
    style={
      Object {
        "height": "100vh",
      }
    }
  >
    <div
      onMouseOver={[Function]}
      style={
        Object {
          "marginLeft": "0px",
          "marginTop": "0px",
          "padding": "40px",
          "transition": "margin 800ms cubic-bezier(0.23, 1, 0.32, 1) 100ms",
        }
      }
    >
      <button
        onClick={[Function]}
        style={
          Object {
            "padding": "5px",
            "position": "relative",
          }
        }
      >
        Main Button
      </button>
    </div>
  </div>
</div>
`;
